private/model/cli/gen-api/main.go

```diff
@@ -135,14 +138,14 @@ func main() {
 			PackageDir: pkgDir,
 		}
 
-		wg.Add(1)
-		go func() {
-			defer wg.Done()
-			writeServiceFiles(g, pkgDir)
-		}()
+		//wg.Add(1)
+		//go func() {
+		//	defer wg.Done()
+		writeServiceFiles(g, pkgDir)
+		//}()
```

Comment:
commented code still needed? Looks like this switches from parallel to sequential generation. I don't have an issue with that change, but can delete old code since not needed anymore.

Reply:
Whoops didn't mean to commit that, was just for doing some debugging.